usersprompt.js

```diff
@@ -0,0 +1,31 @@
+
+var readline = require('readline');
```

Comment:
Great, You can now merge this.
You'll have to make another PR to write tests for the `logAgeGroup` function